django_dbq/management/commands/queue_depth.py

```diff
@@ -0,0 +1,20 @@
+from django.core.management.base import BaseCommand
+from django_dbq.models import Job
+
+
+class Command(BaseCommand):
+
+    help = "Print the current depth of the given queue"
+
+    def add_arguments(self, parser):
+        parser.add_argument("queue_name", nargs="?", default="default", type=str)
+
+    def handle(self, *args, **options):
+        queue_name = options["queue_name"]
+        queue_depths = Job.get_queue_depths_dict()
+
```

Comment:
Worth writing out the total queue depth also?

Reply:
yeah that's a good idea.